### DISCUSSION TITLE
gatsby-theme-jodie Github Pages Site Link

### DISCUSSION THREAD

**Discussion by paul-matthew** [LekoArts/gatsby-themes#1023] [Questions]
I am using gatsby-theme-jodie, I have everything set up and working with the localhost.  But when I preview via github pages site link it is only displaying the readme file.  I understand that this file seems to be the default if there is no index.html.  But I am not sure what to do with this particular theme.  Sorry for the basic question, I am new to this environment

**Comment by LekoArts** [comment]
If you want to use GitHub Pages you'll want to follow these instructions: https://www.gatsbyjs.com/docs/how-to/previews-deploys-hosting/how-gatsby-works-with-github-pages/
